directory/directory_dest.go

```diff
@@ -15,13 +16,10 @@ import (
 	"github.com/containers/image/v5/internal/putblobdigest"
 	"github.com/containers/image/v5/internal/signature"
 	"github.com/containers/image/v5/types"
-	"github.com/containers/storage/pkg/fileutils"
 	"github.com/opencontainers/go-digest"
 	"github.com/sirupsen/logrus"
 )
 
-const version = "Directory Transport Version: 1.1\n"
```

Comment:
The way I think about it, this is really a constant of the file format, not a private implementation detail of `initDestDir` (we can’t just change the contents to make `initDestDir` individually nicer, for example); so it should be a global constant in the module. Also, eventually, https://github.com/containers/image/issues/1876 will need this to live outside of `initDestDir`.

[Well, we should eventually actually parse the numbers and do comparisons, but that’s not yet necessary.]